### PR TITLE
Ll insertions

### PR DESCRIPTION
**Pull request by mparkersimms**
- [ x] Top-level README “Table of Contents” is updated
 - [x ] Feature tasks for this challenge are completed
 - [ x] Unit tests written and passing
     - [ x] “Happy Path” - Expected outcome
     - [x ] Expected failure
     - [ x] Edge Case (if applicable/obvious)
 - [ x] README for this challenge is complete
     - [ x] Summary, Description, Approach & Efficiency, Solution
     - [x ] Link to code
     - [x ] Picture of whiteboard

### CODE REVIEW

File: linkedlist/app/src/test/java/linkedlist/AppTest.java [hunk 1]
```diff
@@ -34,5 +34,44 @@
         String testString = "{Do},{you},{like},{this},{test?},";
         assertEquals("The linked list should return a string of the items in the list", testString,linkedList.toString() );
     }
+    @Test public void appendTest(){
```
Comment (Mattpet26) [inline linkedlist/app/src/test/java/linkedlist/AppTest.java:37]: Good work on your tests. Each method should have it's own test!
